.github/workflows/maven.yml

```
@@ -1,38 +1,37 @@
-# This workflow will build a Java project with Maven
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
-
 name: Build Test
-
-on: [push, pull_request]
+on:
+  # Build PRs and branches.
+  pull_request:
+    # TODO: Uncomment this before merging.
+    #paths-ignore:
+    #  - .github/workflows/deploy-tagged.yml
```

This should be removed before merging - was commented out to allow builds to trigger, because the current PR contains changes to the `deploy-tagged.yml` file.
```suggestion
    paths-ignore:
      - .github/workflows/deploy-tagged.yml
```